src/inmanta/server/extensions.py

```diff
@@ -124,7 +128,7 @@ def _load_feature_config(self) -> dict[str, dict[str, Any]]:
             return {}
 
         with open(feature_file, encoding="utf-8") as fd:
-            result = yaml.safe_load(fd)
+            result: dict[str, dict[str, dict[str, FeatureValueTypes]]] = yaml.safe_load(fd)
```

Comment:
This isn't type safe, is it? Not that it was before, so not really an objection.

Reply:
I know and this code path is actually never used because we load the data from the license in inmanta-license.

For me this serves mostly as a reminder what the structure is that we load